src/amo/components/CollectionSort/styles.scss

```diff
@@ -1,12 +1,9 @@
 @import '~photon-colors/photon-colors';
 
-.CollectionSort {
-  margin-top: 12px;
-}
-
 .CollectionSort-label {
   color: $grey-50;
   display: block;
+  padding-bottom: 6px;
 }
 
 .CollectionSort-select {
```

Comment:
The old code had `margin: 0 0 6px` but you can delete that entirely. You can just rely on the card padding. Here you will see that the extra margin causes the layout to look lopsided:

<img width="679" alt="screenshot 2018-07-24 14 24 09" src="https://user-images.githubusercontent.com/55398/43161541-fcaaa122-8f4d-11e8-92de-e465659302d9.png">
<img width="348" alt="screenshot 2018-07-24 14 28 14" src="https://user-images.githubusercontent.com/55398/43161544-fe9ee7f4-8f4d-11e8-9a53-e0f2523584e3.png">

Reply:
btw, I was able to use the Redux devtool to travel back in time to this state. However, the auto search input breaks when you time travel since the third party component we use is implemented with `setState()`.